File: URI Programs/C/1046.c
#include <stdio.h>

int main(){
	int initial, final, duration;
	scanf("%d %d", &initial, &final);
	if (initial >= final){
		duration = 24 - initial + final;
	} else {
		duration = final - initial;
	}
	printf("O JOGO DUROU %d HORA(S)\n", duration);
	return 0;
}

File: URI Programs/Python/1046.py
x, y = map(int, input().split())

if x == y:
  print("O JOGO DUROU 24 HORA(S)")
elif x > y :
  x = 24 - x
  h = x + y
  print("O JOGO DUROU "+str(h)+" HORA(S)")
elif x < y:
  h = y - x
  print("O JOGO DUROU "+str(h)+" HORA(S)")